_templates/how-to.md

```diff
@@ -1,26 +1,72 @@
 ---
 title: [Character Limit: 255, Recommendation is to have <= 75 characters for a title as this can impact indexing by search engines like Google, Bing, etc. The article title should be a combination of the main problem statement and if applicable, the key environment statement. Also determines the title in the table of contents.]
-description: [Brief description of the How-To article]
-type: [HowTo]
-page_title: [Text that will populate the HTML file’s head:title attribute]
+description: [Character Limit: 255, Recommendation is to have <= 155 characters for a description as this can impact indexing by search engines like Google, Bing, etc. The description should be brief overview of the how-to article. It should exclude unnecessary phrases like "This article contains ..."]
+type: [how-to]
+page_title: [Text that will populate the HTML file’s head:title attribute. Normally, this should be the same as the title field.]
 slug: [Unique identifier of the article. Used for cross-referencing. Ex: how-to-write-an-essay-nativescript]
-position: [Integer specifying the display position of the article in the navigation. If omitted, the articles are ordered alphabetically based on file name]
+position: [Keep this field blank for now!!! Integer specifying the display position of the article in the navigation. If omitted, the articles are ordered alphabetically based on file name]
 tags: [Relevant keywords. Currently not used but always fill in for future proofness]
-<!-- TO DO: Do we we need the below fields for How-To articles? -->
-teampulseid: [number]
-ticketid: [number]
-pitsid: [number] <!-- Not sure we need this, as pits item are related to teampulse anyway -->
+teampulseid: [A number, representing the TeamPulse ID of the issue\question. Leave this blank if the issue\question doesn't exist as an item in TeamPulse]
+ticketid: [A number, representing the ticket ID associated with this issue\question. The field can contain more than one ticket IDs (e.g. several clients have reported it). Leave this blank if the issue\question is not related to a support case]
```

Comment:
The field can contain more than one ticket IDs (e.g. several clients have reported it) -> Perhaps we should agree on a convention about separating multiple items. E.g. "separated with a comma, separated with a space" or something else.